lib/shopify_cli/command.rb

```diff
@@ -110,7 +110,11 @@ def check_node_version
       def check_version(version, range:, runtime:)
         return if Environment.test?
         return if range.nil?
-        return if version >= range.from && version < range.to
+
+        version_without_pre_nor_build = Utilities.version_dropping_pre_and_build(version)
+
+        return if version_without_pre_nor_build >= Utilities.version_dropping_pre_and_build(range.from) &&
+        version_without_pre_nor_build < Utilities.version_dropping_pre_and_build(range.to)
```

Comment:
I'm amazed Rubocop isn't complaining about a 2-line `return if` but it definitely isn't my cup of 🍵